File: app/src/main/java/vn/loitp/app/activity/customviews/button/roundedbutton/RoundedButtonActivity.java
package vn.loitp.app.activity.customviews.button.roundedbutton;

import android.os.Bundle;
import android.view.View;

import com.core.base.BaseFontActivity;
import com.views.LToast;

import loitp.basemaster.R;

public class RoundedButtonActivity extends BaseFontActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        View.OnClickListener onClickListener = view -> LToast.showShort(getActivity(), "Click");
        findViewById(R.id.bt_0).setOnClickListener(onClickListener);
        findViewById(R.id.bt_1).setOnClickListener(onClickListener);
        findViewById(R.id.bt_2).setOnClickListener(onClickListener);
        findViewById(R.id.bt_3).setOnClickListener(onClickListener);
    }

    @Override
    protected boolean setFullScreen() {
        return false;
    }

    @Override
    protected String setTag() {
        return getClass().getSimpleName();
    }

    @Override
    protected int setLayoutResourceId() {
        return R.layout.activity_rounded_button;
    }
}

File: app/src/main/java/vn/loitp/app/activity/customviews/button/roundedbutton/RoundedButtonActivity.kt
package vn.loitp.app.activity.customviews.button.roundedbutton

import android.os.Bundle
import com.annotation.LayoutId
import com.annotation.LogTag
import com.core.base.BaseFontActivity
import kotlinx.android.synthetic.main.activity_button_rounded.*
import vn.loitp.app.R

@LayoutId(R.layout.activity_button_rounded)
@LogTag("RoundedButtonActivity")
class RoundedButtonActivity : BaseFontActivity() {

    override fun setFullScreen(): Boolean {
        return false
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        bt0.setOnClickListener { showShort(getString(R.string.click)) }
        bt1.setOnClickListener { showShort(getString(R.string.click)) }
        bt2.setOnClickListener { showShort(getString(R.string.click)) }
        bt3.setOnClickListener { showShort(getString(R.string.click)) }
    }
}
